charts/redpanda/templates/cert-issuers.yaml

```diff
@@ -65,6 +66,7 @@ metadata:
   {{- . | nindent 4 }}
 {{- end }}
 spec:
+  duration: {{ $d | default "43800h" }}
```

Comment:
The default is 90d if unset right? Why 5 years?

Reply:
That's what the server cert defaults to, if not overridden. Since this is the CA that signs that cert, having the it expire (long) before the server certs doesn't make a lot of sense, and could lead to so some weird behaviors (e.g. some clients continuing to work and others rejecting it). Also, having a very long duration for the root CA cert seems seems to be common practice and makes sense intuitively -- trusted CA sets are often baked into images, applications, etc. and aren't always easy to manage.